assembly/node.d.ts

```diff
@@ -14,3 +14,14 @@ declare class Buffer extends Uint8Array {
   /** Reads a signed integer at the designated offset. */
   readInt8(offset?: i32): i8;
 }
+
+declare namespace Buffer {
+  export namespace HEX {
+    /** Creates an ArrayBuffer from a given string that is encoded in the HEX format. */
```

Comment:
According to [wikipedia](https://en.wikipedia.org/wiki/Hexadecimal) hex is lowercase.

Reply:
This should be all set. (we are following std lib formatting, so it's good as it is.)